CRM/Member/Import/Form/MapField.php

```diff
@@ -288,68 +288,66 @@ public function buildQuickForm() {
   public static function formRule($fields, $files, $self) {
     $errors = [];
 
-    if (!array_key_exists('savedMapping', $fields)) {
```

Comment:
Remember to use w=1 to make this readable....https://github.com/civicrm/civicrm-core/pull/23283/files?w=1